### DISCUSSION TITLE
Tokens used by 1 conversation - anthropic

### DISCUSSION THREAD

**Discussion by luciannuta** [danny-avila/LibreChat#4368] [Q&A]
Hello, does having a longer conversation make it become increasingly costly because it always reads the previous messages? I feel like this is how anthropic api worked for me

**Comment by danny-avila** [comment]
Yes but prompt caching should help with this.

**Reply by luciannuta** [reply]
yes, but it looks like prompt caching was activated

**Comment by luciannuta** [comment]
![image](https://github.com/user-attachments/assets/92f5bfec-9ef3-4cc6-baf0-3432ec909013)
And using max context tokens leads to "Something went wrong. Here's the specific error message we encountered: Operation failed after 3 attempts: 400 {"type":"error","error":{"type":"invalid_request_error","message":"messages: at least one message is required"}}"
